scripts/opt/backup-loop.sh

```diff
@@ -619,6 +621,9 @@ while true; do
       log ERROR "Unable to turn saving off. Is the server running?"
       exit 1
     fi
+  elif [[ $first_run == TRUE && ${BACKUP_ON_STARTUP^^} = FALSE ]]; then
+    log INFO "Skipping backup on startup"
+    first_run=false
```

Comment:
I'm not sure if this is the right place for the conditional. This is for the case when the server is auto-paused. 

Won't the backup call on line 605 happen still?

Reply:
In practice, this works for me as I expected. I'll take another look if I can figure out a better option.

Reply:
Please provide your compose file or equivalent used for testing.